tools/hooks/restrict_levelbuilder_changes.rb

```diff
@@ -8,6 +8,8 @@
   dashboard/config/locales/scripts.en.yml
   dashboard/config/locales/courses.en.yml
   dashboard/config/locales/unplugged.en.yml
+  dashboard/config/locales/data.en.yml
+  dashboard/config/videos.csv
```

Comment:
👍

Reply:
Will thumbnails for these changes also get created on levelbuilder?
I guess it's fine to follow up with that since LB always gets merged into staging.